### DISCUSSION TITLE
very big .git folder in /opt/homebrew/Library/Taps/homebrew/homebrew-core/.git

### DISCUSSION THREAD

**Discussion by n-osennij** [Homebrew/discussions#2994] [Everyday usage]
How can I delete this folder (511 mb) on `/opt/homebrew/Library/Taps/homebrew/homebrew-core/.git`?

**Comment by SMillerDev** [comment]
No, homebrew uses that folder to keep track of software revisions.

**Comment by hytor-yang** [comment]
Set `HOMEBREW_INSTALL_FROM_API` environment variable and you can safely get rid of `.git` from both homebrew/core and home/cask (by untap command).

See Homebrew/brew#12556 .
